modules/ROOT/pages/building-soap-proxy.adoc

```diff
@@ -81,6 +81,44 @@ You can resolve this issue by adding the appropriate imports in the WSDL file. F
 </types>
 ----
 
+== Retrieving a WSDL using Basic Authentication credentials
+
+The SOAP proxy provides a mechanism for retrieving a WSDL hosted on a remote site protected with Basic Auth. This is achieved by configuring the application properties `basic.auth.user` and `basic.auth.password`.
+
+These properties are set in the `basicauth.properties` config file and passed to the validator module.
+
+[source,xml,linenums]
+----
+<soap-request-validator:config name="validator-config" wsdlLocation="http://myremoteurl:9092/service?wsdl" serviceName="myServiceName"
+                                   servicePort="myServicePort" schemaValidationEnabled="true"
+                                   basicAuthUser="${basic.auth.user}" basicAuthPassword="${basic.auth.password}"/>
+----
+
+If no value is provided for the properties, then the WSDL retrieval is done without Basic Authentication.
+
+== Retrieving a WSDL using TLS
+
+In order to retrieve a WSDL using a TLS context, then the validator module present in the proxy must be configured using the `requesterConfig` parameter.
```

Comment:
This section is exposing internal implementation that the customer does not care. We should just state that starting from version blabla we support OOTB the TLS Scenario by reusing configuration set in APIM